azext_edge/tests/edge/orchestration/test_instance_unit.py

```diff
@@ -104,3 +109,58 @@ def test_instance_list(mocked_cmd, resource_group_name: str, records: int):
     instances = Instances(mocked_cmd)
     result = list(instances.list(resource_group_name=resource_group_name))
     assert result == mock_instance_records["value"]
+    assert len(mocked_responses.calls) == 1
+
+
+@pytest.mark.parametrize(
+    "description",
+    [None, generate_random_string()],
+)
+@pytest.mark.parametrize(
+    "tags",
+    [None, {"a": "b", "c": "d"}],
+)
+def test_instance_update(mocked_cmd, mocked_responses: responses, description: Optional[str], tags: Optional[dict]):
+    instance_name = generate_random_string()
+    resource_group_name = generate_random_string()
+
+    mock_instance_record = get_mock_instance_record(name=instance_name, resource_group_name=resource_group_name)
+    mocked_responses.add(
+        method=responses.GET,
+        url=f"{BASE_URL}/subscriptions/{ZEROED_SUBSCRIPTION}/resourceGroups/{resource_group_name}/"
```

Comment:
seeing this multiple times, could use a constant string (that would be formatted)

Reply:
Constant string didn't make sense due to the variations but took the feedback and made a utility function.